src/Parser/GTXParser.js

```diff
@@ -0,0 +1,73 @@
+ * GTX file in, and returns a `{@link GeoidGrid}`. The `{@link GeoidGrid}` contains all the necessary attributes and
+ * methods to access the GTX data in iTowns.
+ *
+ * @module GTXParser
+ */
+export default {
+    /**
+     * Parses a GTX file content and returns a corresponding `{@link GeoidGrid}`.
+     *
+     * @param   {ArrayBuffer}   gtx                             The content of the GTX file to parse.
+     * @param   {Object}        options                         An object gathering the optional parameters to pass to
+                                                                * the parser.
+     * @param   {Object}        [options.in={}]                 Information on the GTX data.
+     * @param   {string}        [options.in.crs='EPSG:4326']    The Coordinates Reference System (CRS) of the GTX data.
+                                                                * It must be a geographic CRS, and must be given as an
+                                                                * EPSG code.
+     * @param   {string}        [options.in.dataType='float']   The encoding of geoid height data within the GTX file.
+                                                                * Must be `'float'` or `'double'`.
+     *
+     * @returns {Promise<GeoidGrid>}    A promise resolving with a `{@link GeoidGrid}`, which contains all the necessary
+                                        * attributes and methods to access GTX file data.
+     */
+    parse(gtx, options = { in: {} }) {
+        const dataType = options.in.dataType || 'float';
+        if (!['float', 'double'].includes(dataType)) {
+            throw new Error(
+                '`dataType` parameter is incorrect for GTXParser.parse method. ' +
+                'This parameter must be either `double` or `float`.',
+            );
+        }
+
+        const headerView = new DataView(gtx, 0, 40);
+        const header = {
+            minX: headerView.getFloat64(8),
+            // formula for the max longitude : maxLongitude = minLongitude + deltaLongitude * (nColumns - 1)
+            maxX: headerView.getFloat64(8)
+                + headerView.getFloat64(24) * (headerView.getInt32(36) - 1),
+            minY: headerView.getFloat64(0),
+            // formula for the max latitude : maxLatitude = minLatitude + deltaLatitude * (nRows - 1)
+            maxY: headerView.getFloat64(0)
+                + headerView.getFloat64(16) * (headerView.getInt32(32) - 1),
+            stepX: headerView.getFloat64(24),
+            stepY: headerView.getFloat64(16),
+            nColumns: headerView.getInt32(36),
+        };
+
+        const data = new DataView(gtx, 40);
+
+        return Promise.resolve(new GeoidGrid(
+            new Extent(
+                options.in.crs || 'EPSG:4326',
+                header.minX, header.maxX, header.minY, header.maxY,
+            ),
+            { x: header.stepX, y: header.stepY },
```

Comment:
could use `const step = Vector2()` and pass to `GeoidGrid`

Reply:
I changed this part accordingy.